packages/ui2/src/components/input.tsx

```diff
@@ -0,0 +1,26 @@
+import * as React from "react";
+
+import { cn } from "../lib/utils.js";
+
+export interface InputProps
+  extends React.InputHTMLAttributes<HTMLInputElement> {}
+
+const Input = React.forwardRef<HTMLInputElement, InputProps>(
```

Comment:
In all the components you created in this PR, no need to use `React.forwardRef` now because we are using React 19:

https://react.dev/blog/2024/12/05/react-19#improvements-in-react-19

Reply:
The button component I created and you modified in this PR doesn't use `forwardRef`, for example.

Reply:
updated, thanks